### PR TITLE
Siris

### PR DESCRIPTION
**Pull request by swiftrax**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/ffkeebs/siris/siris.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "siris.h"
```
Comment (drashna) [inline keyboards/ffkeebs/siris/siris.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/ffkeebs/siris/info.json [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+            },
+            {
+               "label":"1,4",
+               "x":4.5,
+               "y":1.125
+            },
+            {
+               "label":"1,7",
+               "x":12.75,
+               "y":1.1300000000000001
```
Comment (fauxpark) [inline keyboards/ffkeebs/siris/info.json:93]:
```suggestion
               "y":1.13
```

File: keyboards/ffkeebs/siris/info.json [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+            },
+            {
+               "label":"1,9",
+               "x":14.75,
+               "y":1.1300000000000001
```
Comment (fauxpark) [inline keyboards/ffkeebs/siris/info.json:98]:
```suggestion
               "y":1.13
```

File: keyboards/ffkeebs/siris/info.json [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+            },
+            {
+               "label":"1,1",
+               "x":1.5,
+               "y":1.375
+            },
+            {
+               "label":"1,0",
+               "x":0.5,
+               "y":1.3800000000000001
```
Comment (fauxpark) [inline keyboards/ffkeebs/siris/info.json:118]:
```suggestion
               "y":1.38
```

File: keyboards/ffkeebs/siris/info.json [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+            },
+            {
+               "label":"1,10",
+               "x":15.75,
+               "y":1.3800000000000001
```
Comment (fauxpark) [inline keyboards/ffkeebs/siris/info.json:123]:
```suggestion
               "y":1.38
```

File: keyboards/ffkeebs/siris/info.json [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+            },
+            {
+               "label":"1,11",
+               "x":16.75,
+               "y":1.3800000000000001
```
Comment (fauxpark) [inline keyboards/ffkeebs/siris/info.json:128]:
```suggestion
               "y":1.38
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**